server/models/schema.sql

```diff
@@ -1,3 +1,4 @@
+drop database bhima_test; create database bhima_test; use bhima_test;
```

Comment:
This breaks Travis, which infers the database name from the command line.  It also restricts developers from using database names other than `bhima_test`.  Could you remove these lines?